docs/src/pages/[platform]/components/selectfield/react.mdx

```diff
@@ -175,6 +177,42 @@ A disabled field will not be focusable or mutable, and it will not be submitted
   </ExampleCode>
 </Example>
 
+### Multiple state
```

Comment:
```suggestion
### Multiple selections
```
nit: Maybe a different heading? Since we're not really doing anything with state.